host/sgx/exception_sim.c

```diff
@@ -0,0 +1,449 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/registers.h>
+#include <openenclave/internal/sgx/td.h>
+#include <signal.h>
+#if defined(_WIN32)
+#else
+#include <ucontext.h>
+#endif
+#include "enclave.h"
+#include "exception.h"
+
+#if defined(_WIN32)
+bool is_simulate(struct _EXCEPTION_POINTERS* exception_pointers)
```

Comment:
"is simulate" seems like bad grammar to me

```suggestion
bool is_simulation(struct _EXCEPTION_POINTERS* exception_pointers)
```

For comparison, sgx\create.c has: 
`enclave->simulate = oe_sgx_is_simulation_load_context(context);`
and sgx\sgxload.c has:
`bool is_simulation`
both of which use "is simulation" not "is simulate"

Reply:
Suggestion is applied, thanks!